src/views/OrderList.vue

```diff
@@ -3,7 +3,7 @@ import { ref, computed, onMounted } from 'vue';
 import OrderItemRow from '../components/OrderItemRow.vue';
 import { products } from '../util/constants';
 import { db } from '../fb.js';
-import { collection, query, getDocs, orderBy, updateDoc, doc } from "firebase/firestore";
+import { collection, query, getDocs, orderBy, updateDoc, doc, onSnapshot } from "firebase/firestore";
```

Comment:
Consider removing the onShapshot method import, since it is no longer being used.

Reply:
ok